### DISCUSSION TITLE
Video Assistance

### DISCUSSION THREAD

**Discussion by Creep44** [DIGITALCRIMINALS/OnlyFans#400] [Q&A]
I tried following the instructions like 50 times and I am still confused and stuck, Can someone send a video of them setting up the onlyfans and the fansly scraper from scratch. This would definitely help out multiple people who come across this, and are confused as me.

**Comment by roweger** [comment]
You are more likely to receive help if you're more specific about what exactly is the trouble you're experiencing and at what point you experience it, and what exactly you have tried. Few people are going to be willing to take the time to hold your hand to the level you're requesting.

Are you saying you are receiving an error message, or you just literally don't understand the installation instructions?
